x/dex/keeper/msg_server_limit_test.go

```diff
@@ -363,25 +368,25 @@ func (s *MsgServerTestSuite) TestLimitOrderPartialFillDepositCancel() {
 
 	s.bobMarketSells("TokenA", 10, 10)
 
-	s.assertAliceBalances(100, 40)
-	s.assertBobBalances(80, 120)
-	s.assertDexBalances(20, 40)
+	// s.assertAliceBalances(100, 40)
```

Comment:
Commented out bit seems to be passing... Why is it disabled?

Reply:
should reenable, thanks